src/toil/jobStores/googleJobStore.py

```diff
@@ -256,7 +256,7 @@ def writeFileStream(self, jobStoreID=None, cleanup=False, basename=None):
 
     def getEmptyFileStoreID(self, jobStoreID=None, cleanup=False, basename=None):
         fileID = self._newID(isFile=True, jobStoreID=jobStoreID if cleanup else None)
-        self._writeFile(fileID, StringIO(""))
+        self._writeFile(fileID, BytesIO(bytes("", 'utf-8')))
```

Comment:
I also want to add, you should only need `BytesIO(b"")` here.

Note: You're converting from unicode with this and it's equivalent to saying that the unicode string has encoding `utf-8`, please convert it to a raw bytestring based on that encoding:

`""` -> `bytes("", 'utf-8')` -> `b""`